app/views/exchanges/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<nav class="navbar" style="background-color: #3778c2;" role="navigation">
+  <div class="navbar-brand">
+    <a href="/" class="navbar-item">
```

Comment:
- [ ] When a user is on the add new transaction form page, and they click the arrow "back" button (<) they should be taken to the transactions index page of the category, not the category page or root page.

  ![image](https://github.com/AlyAtalla/budget-app/assets/54848363/fb546404-f2df-4068-bb60-1d0c1dca4a56)